### PR TITLE
Update the linting setup

### PR DESCRIPTION
**Pull request by Windvis**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# unconventional js
+/blueprints/*/files/
+/vendor/
+
+ # compiled output
```
Comment (bertdeblock) [inline .prettierignore]: Seems to be a leading space here.

File: package.json [hunk 2]
```diff
@@ -27,10 +28,12 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.1.2"
+    "ember-cli-babel": "^7.1.2",
+    "husky": "^1.3.1",
```
Comment (bertdeblock) [inline package.json]: mofo

Comment (Windvis) [reply package.json]: I. M. DERP.

Comment (Windvis) [reply package.json]: fikst

File: .prettierignore (hunk 1 above)
Comment (Windvis) [reply .prettierignore]: dees ok

**Pull request merged**